src/app/calendar/page.tsx

```diff
@@ -1,14 +1,26 @@
 import { Metadata } from "next";
 import React from "react";
 import ParcelCalendar from "@/app/calendar/ParcelCalendar";
-import getParcelCalendarEvents from "@/app/calendar/getParcelCalendarEvents";
+import {
+    ClientMap,
+    getClientMap,
+    getParcelsWithCollectionDate,
+} from "@/app/calendar/parcelCalendarFunctions";
+import { Schema } from "@/supabase";
 
 const CalendarPage = async (): Promise<React.ReactElement> => {
-    const events = await getParcelCalendarEvents();
+    const [clientMap, parcelsWithCollectionDate]: [ClientMap, Schema["parcels"][]] =
+        await Promise.all([
+            Promise.resolve(getClientMap()),
+            Promise.resolve(getParcelsWithCollectionDate()),
```

Comment:
I don't think `Promise.resolve` does what you think it does, you don't need it here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/resolve

Reply:
Ah right removed